mini_portile2.gemspec

```diff
@@ -40,4 +40,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "minitest-hooks", "~> 1.5"
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "webrick", "~> 1.7"
+
+  spec.metadata["changelog_uri"] = spec.homepage + "/blob/master/CHANGELOG.md"
```

Comment:
Needs to be `main` and not `master` to match the URL of the changelog at https://github.com/flavorjones/mini_portile/blob/main/CHANGELOG.md

Reply:
Ugh, apologies @flavorjones, how careless of me. Thank you for spotting it. Fixed in [4390ac4](https://github.com/flavorjones/mini_portile/pull/147/commits/4390ac4d132f16d5669cc404fe6bb79c0bd48977)